src/features/instance/apis/requestSnippets.ts

```diff
@@ -0,0 +1,8 @@
+export const requestSnippets = {
+	generators: {
+		node_fetch: {
+			title: 'Node.js Fetch',
```

Comment:
Just `fetch`. There isn't any node.js specific about this, I don't think.

Reply:
Done.